@types/ReactNative.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+export * from "react-native";
+declare module "react-native" {
```

Comment:
Why react-native? Shouldn't this be our own module?

Reply:
This essentially extends react-native's NativeModule; so we can import our APIClient and GenericClient with types / autocomplete / IDE goodness.